src/main/java/com/marek/onlinebookstore/dto/order/OrderDto.java

```diff
@@ -0,0 +1,21 @@
+package com.marek.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public record OrderDto(
+        @Positive
+        Long id,
+        @Positive
+        Long userId,
+        Set<OrderItemDto> orderItems,
+        LocalDateTime orderDate,
```

Comment:
The 'orderDate' field should be read-only and not included in the DTO for client requests. It should be set on the server side when the order is placed.